docs/Makefile

```diff
@@ -21,26 +21,36 @@ endif
 
 all: deps api build
 
-api:
-	$(QUARTODOC) build
-	$(QUARTODOC) interlinks
+ensure-dev:
+	$(MAKE) -C .. dev
+
+api: ensure-dev
+	@echo "::group::quartodoc interlinks"
+	$(UV) run $(QUARTODOC) interlinks
+	@echo "::endgroup::"
+	@echo "::group::quartodoc build"
+	$(UV) run $(QUARTODOC) build --verbose
+	@echo "::endgroup::"
 	cp -r _extensions/ reference/_extensions  # Required to render footer
 
-build:
-	CURRENT_YEAR=$(CURRENT_YEAR) \
-	PROJECT_VERSION=$(PROJECT_VERSION) \
-	$(QUARTO) render
+build: ensure-dev
+	source ../.venv/bin/activate && \
```

Comment:
Should this utilize `$(MAKE) -C .. ensure-uv` instead?

Reply:
The environment has to be active. Maybe we can get it to work with `$(UV) tool run --with ../ $(QUARTO) render`?

Reply:
The `$(MAKE) -C .. ensure-uv` is performed within the `ensure-dev` target (so we'd only need to call it once for multiple targets)